### ISSUE TITLE
 fork: Resource temporarily unavailable

### ISSUE THREAD

**Issue by 0xdevalias** [ctran/annotate_models#587]
Receiving ` fork: Resource temporarily unavailable` when trying to run annotate.

Googling this sounds like it is more of a system issue.. but the results (and solutions) I am seeing are set way lower than my system:

* eg. http://blog.ghostinthemachines.com/2010/01/19/mac-os-x-fork-resource-temporarily-unavailable/

My system:
```
⇒  sysctl -a | grep maxproc
kern.maxproc: 2128
kern.maxprocperuid: 1418
```

So i'm wondering if there might be some way I can pass additional flags to annotate that might avoid hitting this.

## Commands

```
$ annotate --show-migration --simple-indexes --classified-sort --exclude tests,fixtures,factories,serializers --format markdown
./bin/annotate: fork: Resource temporarily unavailable
```

## Version

- annotate version: `v2.7.2`
- rails version: `5.1.6.1`
- ruby version: `2.3.6p384`

**Comment by 0xdevalias** [comment]
So turns out I had `./bin` in my path, and a wrapper script in `./bin/annotate`. Queue recursive calls until resources run out with seemingly unhelpful (but entirely accurate) error message.

tl;dr: don't add `./bin` to your `PATH`
